src/components/Dialogs/elements/DialogForm.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <form class="form">
+        <more-options
+            v-show="options && condition"
```

Comment:
why show instead of if?

Reply:
When I started to learn vue I learnt I should always prefer `v-show` over `v-if` because `v-show` only hides an element whereas `v-if` takes it out from the DOM and you achieve a better performance when doing the first thing.

But recently I actually started to think about that 😛 and now I believe there is no reason for having many hidden elements in the DOM, so I'm not sure yet but I think `v-if` can be actually a better option...

**Corrected**

Reply:
`v-if` also mounts and unmount the component, so the whole logic behind unrendered component won't be loaded until it's mounted (condition turns to true)